vllm/v1/engine/async_llm.py

```diff
@@ -372,7 +372,8 @@ async def do_log_stats(
         scheduler_outputs=None,
         model_output=None,
     ) -> None:
-        logger.debug("Called do_log_stats.")
+        for stat_logger in self.stat_loggers:
```

Comment:
One negative of this design is that now the `AsyncLLM` will not log unless some external object asks it to. This works fine for the API server but not for users that leverage `AsyncLLM` directly

Reply:
@robertgshaw2-redhat very much agree, that's what I was referring to above! Will plan a follow-on for that:
> I don't think responsibility for the periodic log_stats task should be in the API server, we should move it inside the engine client and remove the external EnigneClient.do_log_stats method. But that can be changed in a subsequent PR.